content/download/lts/index.adoc

```diff
@@ -5,7 +5,7 @@ title: LTS Release Line
 :toc:
 
 The weekly Jenkins releases deliver bug fixes and new features rapidly to users and plugin developers who need them.
-But for more conservative users, it's preferable to stick to a release line which changes less often and only receives important bug fixes, even if such a release line lags behind in terms of features.
+But for more conservative users, it's preferable to stick to a release line which changes less often and only receives important bug and security related fixes, even if such a release line lags behind in terms of features.
```

Comment:
not changing the policy here - but it should be obvious that LTS gets security fixed (to some people an BUG and a security issue are different things but a CVE is just a bug that is exploitable :) )  anyway - let us call it out - it should do no harm.